res/presets/my bloody valentine.chowpreset

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Preset name="my bloody valentine" plugin="BYOD" vendor="User" category=""
+        version="1.3.0" preset_file="/home/henry/Sync/ThirdPartyStuff/.clap plugins/BYOD/userpresets/User/my bloody valentine.chowpreset">

```

Comment:
A couple things here...
- Who do you want to be listed as the preset "vendor"? It's totally cool to use your on name (or screen name).
- To follow the naming conventions used in the other preset, we would probably want the category to be "Players".
- For presets that are shipped with the plugin, I usually delete the `preset_file` field, since that will be meaningless on other users' computers.

Here's an example preset file if you want to have a look: https://github.com/Chowdhury-DSP/BYOD/blob/main/res/presets/Superdrag.chowpreset#L3

Reply:
> Who do you want to be listed as the preset "vendor"? It's totally cool to use your on name (or screen name).

oh just list it as yourself for now, for the sake of convenience. put it under Players

> For presets that are shipped with the plugin, I usually delete the preset_file field, since that will be meaningless on other users' computers.

yeah do that lol